pom.xml

```diff
@@ -88,8 +88,8 @@
 								<checkBuildResult>true</checkBuildResult>
 								<treatUndefinedAsFailed>true</treatUndefinedAsFailed>
 								<treatSkippedAsFailed>false</treatSkippedAsFailed>
-								<outputDirectory>${project.build.directory}/cucumber-reports</outputDirectory>
-								<inputDirectory>${project.build.directory}/cucumber-reports</inputDirectory>
+								<outputDirectory>${project.basedir}/target/cucumber-reports</outputDirectory>
+								<inputDirectory>${project.basedir}/target/cucumber-reports</inputDirectory>
```

Comment:
I: The build directory for Python projects is set to the distribution folder; this changing this allows us to target both Cucumber and Behave tests without having to do separate path configurations in the reporting plugin for each testing framework.